### PR TITLE
Fix GetLog function to fetch logs of Pod

### PR DESCRIPTION
**Pull request by Akash4927**
This PR will fix the GetLog function to fetch the logs of Pods.

Signed-off-by: Akash Srivastava <akash.srivastava@openebs.io>

### CODE REVIEW

File: k8s_util/k8s_util.go [hunk 1]
```diff
@@ -340,24 +340,29 @@ func GetLog(podName, namespace string) (string, error) {
...
 		goto use_kubectl
 	}
 
-	defer readCloser.Close()
-	_, err = readCloser.Read(bytes)
-	if err == nil {
-		return string(bytes), nil
+	buf.ReadFrom(readCloser)
+	if Debug {
+		fmt.Println("Log of Pod ", podName, " in Namespace ", namespace, " through API:\n", buf.String())
```
Comment (a4abhishek) [inline k8s_util/k8s_util.go]: Can you include PodName and Namespace in this message?

Comment (a4abhishek) [inline k8s_util/k8s_util.go]: Can you remove spaces from the strings here?

Comment (a4abhishek) [reply k8s_util/k8s_util.go]: And put last buf.String() in new fmt.Println. Remove \n from second last as well.

Comment (Akash4927) [reply k8s_util/k8s_util.go]: Addressed. Thanks !!

**Pull request merged**